3party/opening_hours/rules_evaluation.cpp

```
@@ -108,6 +108,7 @@ osmoh::RuleState ModifierToRuleState(osmoh::RuleSequence::Modifier const modifie
 
     case Modifier::Unknown:
     case Modifier::Comment:
+    default:
```

No, it will silence the error if a new switch case will be added.